File: sem1/Programming Fundamentals/StudentManagement/model/student.py
class Student(object):

    def __init__(self, studentId, name):
        self.__studentId = studentId
        self.__name = name

    def __str__(self):
        return 'id: ' + str(self.__studentId) + ' name: ' + str(self.__name)

    def __eq__(self, other):
        if type(other) == int:
            return self.__studentId == other
        return self.__studentId == other.__studentId

    def getId(self):
        return self.__studentId

    def getName(self):
        return self.__name

    def setName(self, newName):
        self.__name = newName

File: sem4/Systems Design and Implementation/catalog-app/src/main/java/ubb/domain/Student.java
package ubb.domain;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;

@Entity
@Table(name = "student", schema = "public")
public class Student extends BaseEntity<Long>{

    @Column(name = "serial_number")
    private String serialNumber;

    @Column(name = "name")
    private String name;

    @Column(name = "\"group\"")
    private int group;

    public Student() {

    }

    public Student(String serialNumber, String name, int group) {
        this.serialNumber = serialNumber;
        this.name = name;
        this.group = group;
    }

    public String getSerialNumber() {
        return serialNumber;
    }

    public void setSerialNumber(String serialNumber) {
        this.serialNumber = serialNumber;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getGroup() {
        return group;
    }

    public void setGroup(int group) {
        this.group = group;
    }

    @Override
    public String toString() {
        return "Student{" +
                "serialNumber='" + serialNumber + '\'' +
                ", name='" + name + '\'' +
                ", group=" + group +
                "} " + super.toString();
    }
}
